src/pudl/transform/eia861.py

```diff
@@ -2333,7 +2403,6 @@ def core_utility_data_eia861(raw_eia861__utility_data: pd.DataFrame):
 
     Transformations include:
 
-    * Remove rows with utility ids 88888.
```

Comment:
Same as above - probably something like "Aggregate values for rows with utility id 88888 (anonymized) - see _combine_88888_values and _pre_process for details."